modules/ROOT/partials/misc/supported-versions.adoc

```diff
@@ -3,17 +3,18 @@
 
 Supported versions of {productname}:
 
-[cols="^,^,^",options="header"]
+[cols="^,^,^,^",options="header"]
 |===
-|Version |Release Date |End of Support
-|6.7 |2023-09-13 |2025-03-13
-|6.6 |2023-07-19 |2025-01-19
-|6.5 |2023-06-21 |2024-12-21
-|6.4 |2023-03-29 |2024-09-29
-|6.3 |2022-12-08 |2024-06-08
-|6.2 |2022-09-26 |2024-03-26
-|6.1 |2022-07-13 |2024-01-13
-|6.0 |2022-04-07 |2023-10-07
+|Version |Release Date |End of Community Support |End of Premium Support
+|6.8 |2023-12-06 |2024-10-31 |2025-06-06
```

Comment:
Once [this comment](https://github.com/tinymce/tinymce-docs/pull/3284/files#r1591781603) has been verified, we need to update this to match.
```suggestion
|6.8 |2023-12-06 |2024-03-20 |2025-06-06
```

Reply:
This has been confirmed.

Reply:
thanks karl 👍